docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -76,6 +78,35 @@ The optional `{batch}` named query references the Batch of data under test. When
 
 :::
 
+## Source-to-target Expectations
+
+A source-to-target Expectation executes one SQL query for each of two Data Sources and compares their results for equality. This can be helpful for validating consistency between systems during data migration or regular data loading processes. Source-to-target Expectations can detect data drift introduced during the ETL process through discrepancies in schemas, counts, time windows, data types, and precision levels between Data Sources. Here are some examples of comparisons you can test:
+- Every row in table A matches every row in table B.
+- An aggregate metric of table A matches the same aggregate metric of table B.
+- An aggregate metric of table A matches a different aggregate metric of table B. (For example, the count of rows where X is true in table A matches the count of rows where Y and Z are true in table B.)
+
+To compare results for equality, each row returned by the query for the target Data Source will be compared to each row returned by the query for the source Data Source. When you configure a source-to-target Expectation, you set an **Expected percentage of identical rows**. The Expectation will fail if the percentage of identical rows between your two queries falls below this threshold.
+
+The percentage of identical rows is computed by dividing the number of matching rows by the maximum number of rows in either the source result or the target result. Here are some example scenarios:
+
+| Source result row count | Target result row count | Matched rows | Percentage of identical rows |
+| ----------------------- | ----------------------- | ------------ | ---------------------------- |
+| 200                     | 200                     | 200          | 100%                         |
+| 25                      | 100                     | 25           | 25%                          |
+| 100                     | 25                      | 1            | 1%                           |
+| 0                       | 0                       | 0            | 100%                           |
+
+The maximum number of rows that will be returned for comparison from each query is 200. The order of rows returned does not matter unless the number of rows returned would be greater than 200.
```

Comment:
It may be good to offer some guidance here, e.g. "For most cases, you'll have greater than 200 rows, so you'll want your queries to use an `ORDER BY` clause to control the output.